src/main/java/no/entur/antu/config/OAuth2Config.java

```diff
@@ -74,7 +76,7 @@ WebClient orgRegisterWebClient(
       .build()
       .mutate()
       .clientConnector(clientHttpConnector)
-      .defaultHeader("Et-Client-Name", "entur-antu")
+      .defaultHeader("Et-Client-Name", ORGANISATION_ET_CLIENT_NAME)
```

Comment:
"Et-Client-Name" is also used across the code base and can be replaced by a constant.